amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseCoordinator.java

```diff
@@ -324,19 +344,11 @@ public static ShardInfo convertLeaseToAssignment(final Lease lease) {
 
     @Override
     public DynamoDBLeaseCoordinator initialLeaseTableReadCapacity(long readCapacity) {
-        if (readCapacity <= 0) {
-            throw new IllegalArgumentException("readCapacity should be >= 1");
-        }
-        this.initialLeaseTableReadCapacity = readCapacity;
-        return this;
+        throw new UnsupportedOperationException("Please set read capacity using the constructor");
```

Comment:
This is a breaking change.  Keep the setting available.  You can mark them as deprecated though.